mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Realtime.MixProject do
   def project do
     [
       app: :realtime,
-      version: "2.29.10",
+      version: "2.30.0",
```

Comment:
I went with a `minor` bump instead of a `patch` since this was a bit more substantative, but happy to use `2.29.11` if that's better

Reply:
lets go with `2.29.11` for now as all of this changes are non-breaking additive elements